wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -877,6 +880,15 @@ private WorkspaceImpl doCreateWorkspace(WorkspaceConfig config,
                                                      .setAttributes(attributes)
                                                      .setTemporary(isTemporary)
                                                      .build();
+
+        // Register default SSH keypair for this workspace.
+        try {
+            this.sshManager.generatePair(EnvironmentContext.getCurrent().getSubject().getUserId(), "workspace", workspace.getId());
```

Comment:
I think it would better to rewrite this logic by events in ...api-ssh... or che-plugin-machine-ext-server module? Because then ssh module will contain logic related to ssh and workspace module won't contain logic related to ssh :)
@garagatyi WDYT?

Also should we remove default ssh key when workspace is removed?